esmvaltool/recipes/sanity_checks/recipe_create_ranges_obs.yml

```
@@ -0,0 +1,687 @@
+# ESMValTool
+---
+documentation:
+  title: Reasonable variable ranges for sanity checks (OBS)
+  description: >
+    Calculate reasonable variable ranges for sanity checks.
+  authors:
+    - lauer_axel
+    - bock_lisa
+  maintainer:
+    - lauer_axel
+
+
+preprocessors:
+
+  pp_min_pr:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: full
+      statistics:
+        - operator: min
```

```suggestion
    multi_model_statistics:
      span: full
      statistics:
        - operator: min
      keep_input_datasets: false
```

I just noticed that my comment about this disappeared when the CMIP6 recipe was removed. Do you need the output from the individual models here? If not, you can use `keep_input_datasets: false`, this will make the recipe faster since a lot less data needs to be saved.

You can use that for all preprocessors below.